tests/framework/e2e/etcdctl.go

```diff
@@ -698,6 +698,9 @@ func (ctl *EtcdctlV3) Watch(ctx context.Context, key string, opts config.WatchOp
 					var resp clientv3.WatchResponse
 					json.Unmarshal([]byte(line), &resp)
 					if resp.Canceled {
+						if opts.IncludeCancellationResponse {
+							ch <- resp
```

Comment:
Why this change. Please add a comment .

Reply:
the cancellation response was swallowed before and existing tests are not expecting this result. We need to assert that it is returned in: 
https://github.com/etcd-io/etcd/pull/16029/files#diff-d99fed41bb3f2962001bb422375c286f00cf1e1e9f57d8bd596f11bde187dea7R399-R405

Reply:
I think this is because etcdctl doesn't print cancel response, just exists.

Reply:
Oh, forgot to comment on this. Not passing cancel response is just oversight when implementing etcdctl struct. This should happen by default. 

@tjungblu Have you tested just passing it by default? Does it cause other test to fail?

Reply:
Doesn't look like making that the default broke anything, I'll make this change